testutil/apiload/config.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	flags "github.com/btcsuite/go-flags"
+)
+
+const (
+	DefaultResultDirectory = "results"
+	DefaultProfilesPath    = "profiles.json"
+	DefaultServer          = "http://localhost:7777"
+)
+
+type config struct {
+	ConfigPath      string `short:"c" long:"config" description:"Path to a custom configuration file."`
+	ResultDirectory string `short:"d" long:"directory" description:"Directory for the result files."`
+	ProfilesPath    string `short:"p" long:"profiles" description:"Path to custom attack profiles."`
+	Server          string `short:"s" long:"server" description:"Target base URL, with protocol."`
```

Comment:
It is worth mentioning that the trailing slash should be ommited from the server, or we should remove it in validation.